model/Operations/Operations.md

```diff
@@ -0,0 +1,22 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Operations
+
+## Summary
+
+The Operations Profile defines fields for describing the business context of the software that cannot (or not yet) directly extracted from the source repository.
+
+## Description
+
+The intention of the Operations Profile is to provide a common base rather for alignment of tool and infrastructure interfaces and development than for exchanging the data between organizations along the software supply chain. The managed information is expected to be mainly kept within the organizations but in some cases might also be necessary input for processing and determining relevant parameters for the later exchange in the supply chain (e.g. Export Control Classification Number).
+
+The information about Applications shall describe facts about the distribution and business context of a product brought into the market, that could consist of several deliverables. There relation between Application and Deliverable can be 1:n.
```

Comment:
```suggestion
The information about Elements shall describe facts about the distribution and business context of a product brought into the market, that could consist of several deliverables. There relation between Elements and Deliverable can be 1:n.
```

Reply:
Applications could be Products, Projects, Packages, etc.   so probably best to stick with Element term which could be any of these.